rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -344,17 +363,18 @@ def valid_data(self):
 
 		return self.input.infos.isValid(self.index)
 
-	def get_dictionary(self):
+	def get_dictionary(self, member_name = None):
 		"""Gets a dictionary with the values of all the fields of this sample
 
 		The dictionary keys are the field names and the dictionary values correspond
 		to each field value. To see how nested types, sequences, and arrays are
 		represented, see :ref:`Accessing the data`.
 
-		:return: A dictionary containing all the fields of the sample.
+		:param str element_name: (Optional) The complex member within the type to obtain.
+		:return: A dictionary containing all the fields of the sample, or the complex type if element_name is supplied.
```

Comment:
..., or the field of a nested member, which can be another complex type, a sequence or an array